docs/pages/access-controls/sso.mdx

```diff
@@ -531,7 +531,8 @@ SAML and OIDC types:
 ## SSO customization
 
 Use the `display` field in an authentication connector to control the appearance
-of SSO buttons in the Teleport Web UI.
+of SSO buttons in the Teleport Web UI. The icons below will show including just the provider 
```

Comment:
I'm a bit confused - what does "show including just the provider name or additional text" mean?